src/core/model.js

```diff
@@ -329,7 +329,8 @@ export default class Model extends Object3D {
     samplers = {},
     parameters = {},
     settings,
-    framebuffer = null
+    framebuffer = null,
+    transformFeedback = null
```

Comment:
Don't we also need to update the model constructor, to pass through varyings parameter to the Program constructor?

Reply:
+1